build.zig

```diff
@@ -84,11 +85,11 @@ fn addTestsFromDir(b: *std.Build, test_step: *std.Build.Step, dir_path: []const
 }
 
 fn setupExamples(b: *std.Build, target: std.zig.CrossTarget, optimize: std.builtin.OptimizeMode) void {
-    const example_step = b.step("test_examples", "Build examples");
-    const example_names = [_][]const u8{ "basic", "bubble_sort", "sleep" };
+    const example_step = b.step("examples", "Build examples");
+    const example_names = [_][]const u8{ "basic", "bubble_sort", "sleep", "linked_list" };
 
     for (example_names) |example_name| {
-        const example = b.addTest(.{
+        const example = b.addExecutable(.{
```

Comment:
Side note to having benchmarks as executables instead of tests: I've had some discussion about this on ziggit (https://ziggit.dev/t/exclude-test-from-being-cached/2275/, https://ziggit.dev/t/how-to-enable-more-logging-and-disable-caching-with-zig-build-test/2654), see also [here on github](https://github.com/ziglang/zig/issues/1010#issuecomment-840259973). While I'm fine with not having benchmarks in tests, I'm unsatisfied with the whole logging / printing thing at the moment. Zig test output is just unreliable IMHO, and not formatted in a clean way. But that should not be our concern here if we have a main fn .